ci-operator/jobs/openshift/origin/openshift-origin-master-presubmits.yaml

```diff
@@ -551,7 +551,6 @@ presubmits:
     decoration_config:
       skip_cloning: true
     labels:
-      ci-operator.openshift.io/prowgen-controlled: "true"
```

Comment:
@stevekuznetsov, is this acceptable to do?  Is there a better way to get ingress debug logging enabled on specific platforms while we work to deflake them?